decidim-core/db/migrate/20170126151123_add_extra_info_to_processes.rb

```diff
@@ -0,0 +1,8 @@
+class AddExtraInfoToProcesses < ActiveRecord::Migration[5.0]
+  def change
+      add_column :decidim_participatory_processes, :domain, :jsonb
+      add_column :decidim_participatory_processes, :end_date, :date
+      add_column :decidim_participatory_processes, :developer_group, :string
+      add_column :decidim_participatory_processes, :scope, :jsonb
```

Comment:
Because both of them will need translations, and as I'm told then they need to be a jsonb fields to be able to contain all locales.